### ISSUE TITLE
Filtering by tag and filter API ignores filter API

### ISSUE THREAD

**Issue by andershagbard** [Shopify/dawn#957]
I was advised by Shopify support to open the issue here. Not sure if this is the best place for this issue, but here it is. This bug exists on all Shopify themes/stores.

If you filter by a tag and filter API, the filter API is ignored. You can see the issue on this URL:

https://dawn-theme-default.myshopify.com/collections/all/handbag?filter.v.price.gte=&filter.v.price.lte=&filter.v.option.color=Black&sort_by=title-ascending

Here it should only show products which have the color option of black, and is tagged with "handbag".

**Comment by bakura10** [comment]
Yes, unfortunately Shopify does not support the two systems at the same time. It is one or the other. Filtering by tag has inherent limitations (not translatable for instance), you should consider filtering by tag as « deprecated ».

what you can do instead is using a filter by product type (assuming you have a product type called handbag for instance).

**Comment by andershagbard** [comment]
Unfortunately this isn't an option. A product type can only contain one value, while tags can contain multiple.

It seems like a bug to me, as it still reads the applied tags _(See the `<title>`)_

**Comment by pinguluk** [comment]
> Unfortunately this isn't an option. A product type can only contain one value, while tags can contain multiple.
> 
> It seems like a bug to me, as it still reads the applied tags _(See the `<title>`)_

They separated [the tags from the storefront filtering](https://shopify.dev/themes/navigation-search/filtering/tag-filtering) when they released 2.0 I think. I've read some users' comments that tags used to be in the storefront filtering as well, but they removed them and they are now used more as an internal filtering for admins in the dashboard.

As for product type, yeah, you can select only one unfortunately and if you try to use the new metafields they provide, as well, you can a text metafield with limited choices, but again, you can only select only 1 variant.

I think the only solutions is to use an App for the search, that allows you to filter by storefront filtering + tags, but they are monthly paid subscriptions and this kinda sucks.
And the other solution, which I'm trying to do, is to remove the storefront filtering from the theme and only use the tags to filter them for everything.
